.gitignore

```diff
@@ -50,14 +44,8 @@ TAGS.*
 \#*\#
 config.mk
 config.stamp
-keywords.md
-lexer.ml
 Session.vim
 src/etc/dl
-tmp.*.rs
-version.md
-version.ml
-version.texi
 .cargo
 !src/vendor/**
 /src/target/
```

Comment:
Same for these two lines: No such dir exists here, but I wasn't sure if something wouldn't create it? I would argue that's a bug though, everything should happen in build/.

Reply:
`target` and `vendor` were created inside `src` when `Cargo.toml` was inside `src`, now it's in the root.